scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -153,23 +182,48 @@ class ScoverageInstrumentationComponent(val global: Global, extraAfterPhase: Opt
     def safeSource(tree: Tree): Option[SourceFile] = if (tree.pos.isDefined) Some(tree.pos.source) else None
 
     def invokeCall(id: Int): Tree = {
```

Comment:
I am not familiar with this, hoping someone with more knowledge can take a look at this one.